libs/wingsdk/test/simulator/simulator.test.ts

```diff
@@ -178,6 +186,250 @@ test("provides raw tree data", async () => {
   expect(treeData).toMatchSnapshot();
 });
 
+describe("in-place updates", () => {
```

Comment:
Add a test that checks retained resource state is retained (i.e. api.url should stay the same if the resource doesn't have to be stopped or updated)

Reply:
done